### ISSUE TITLE
Add Proposed talks page

### ISSUE THREAD

**Issue by SaraVieira** [SaraVieira/awesome-talks#36]
```
I like @rahuldhawani's idea of the community voting on talks. Basically there could be two categories for the talks that are already in awesome status, and one for the ones where they are still pending.
Should they stay in pending state, or should they be removed after a while? Maybe each day/week without an upvote results in a downvote?
```

```
About the proposal feature, I'd say it's a bit overkill to ask for votes from people since the project is rather young. Would it make sense to let all the videos, be added to the list and instead offer a report as not awesome enough thingy?
```

```
A "Propose a talk" page. As the theme of the Awesome Talks is curated talks, let people propose a talk, and other people can vote on it if they find it "Awesome" enough.
But not sure people will vote or not. Your thoughts on this?
```

All these comments can be found on #7 

Add this page as an experiment but this page is right now blocked by #35 

The query is:

```
import { gql } from 'apollo-boost'

export default gql`
  query speakerVideos {
    allVideoses(
      filter: { isPublished: false }
    ) {
      id
      speaker {
        name
      }
      description
      link
      name
      tags {
        name
        id
      }
    }
  }
`
```

**Comment by shyamks** [comment]
Hey @SaraVieira ,
I was looking at contributing to this. How do you think the page would look if it looked similar to the favourite page ?

**Comment by SaraVieira** [comment]
fixed by #106
